docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -0,0 +1,177 @@
+  * Ubuntu 20.04 long-term support (LTS), 64-bit
+
+  .. note::
+     Since the OpenVINO™ 2022.1 release, CentOS 7.6, 64-bit is not longer supported.
+
+.. tab:: Hardware
+
+  Optimized for these processors:
+
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors 
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel Atom® processor with support for Intel® Streaming SIMD Extensions 4.1 (Intel® SSE4.1)
+  * Intel Pentium® processor N4200/5, N3350/5, or N3450/5 with Intel® HD Graphics
+  * Intel® Iris® Xe MAX Graphics
+  * Intel® Neural Compute Stick 2
+  * Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
+
+.. tab:: Processor Notes
+
+  Processor graphics are not included in all processors. 
+  See `Product Specifications`_ for information about your processor.
+  
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.9 are recommended. 
+
+@endsphinxdirective
+
+## Installing OpenVINO Runtime
+
+@sphinxdirective
+
+.. important::
+   Before you start your journey with installation of the Intel® Distribution of OpenVINO™ toolkit, we encourage you to check the :ref:`code samples <code samples>` in C, C++, Python and :ref:`notebook tutorials <notebook tutorials>`, so you could see all the amazing things that you can achieve with our tool.
+
+@endsphinxdirective
+
+### <a name="install-openvino"></a>Step 1: Download and Install the OpenVINO Package
+
+1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit for Linux](https://software.intel.com/en-us/openvino-toolkit/choose-download). You will download two files: 
+   ```sh
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz.sha256
+   ``` 
+2. Open a command prompt terminal window. You can use the keyboard shortcut: Ctrl+Alt+T
+3. Change the directory to where you downloaded the archive files.<br>
+   For example, if you downloaded the files to the current user's `Downloads` directory, use the following command:
+   ```sh
+   cd ~/Downloads/
+   ```
+4. Verify the checksum of the downloaded `sha256` file:
+   ```sh
+   sha256sum <archive name>.tgz.sha256
+   ```
+5. Extract the archive file:
+   ```sh
+   tar xf <archive name>.tgz -C <INSTALL_DIR>
+   ```
+   <br>You should see the installation has started. 
+
+By default, OpenVINO Runtime is installed to the following directory, referred to as `<INSTALL_DIR>` elsewhere in the documentation:
```

Comment:
For archives there is no default install directory. It is just whereever you unpacked it. That may be ~/Downloads, which is not ideal. We could recommend to move the directory to the previous standard places: 

For root or administrator: `sudo mv <archive> /opt/intel` and for regular users: `mv <archive> $HOME/intel`

Reply:
Implemented.